app/views/warehouse_reports/client_details/actives/index.haml

```diff
@@ -6,14 +6,16 @@
     %h1= title
 = render 'filter'
 
-.row
-  .col-sm-12.text-right.mb-2
+.d-flex
+  %p
+    This report was built to provide supporting details to the "Active Clients" section of the Population Dashboards.
+    %br
+    = "The following #{@enrollments.count} clients with #{@enrollments.values.flatten(1).count} enrollments, have at least one day of service between #{@filter.start} and #{@filter.end}."
+    = render 'common/population_warnings'
```

Comment:
Move to just before `render 'warehouse_reports/limited_by_project_note'` for consistency with other reports?